### PR TITLE
Add advertising related player tests

### PR DESCRIPTION
**Pull request by rolandkakonyi**
## Description

These changes add player integration tests to verify advertising-related events are emitted.

## Changes

Added advertising-related player integration tests.

## Checklist
- [x] 🗒 `CHANGELOG` entry - internal changes only, no changelog needed

### CODE REVIEW

File: integration_test/tests/advertisingTest.ts [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+          schedule: adItems,
+        } as AdvertisingConfig;
+        await startPlayerTest({ advertisingConfig }, async () => {
+          await callPlayer(async (player) => {
+            player.load(Sources.artOfMotionHls);
+            player.play();
+          });
+          if (type !== 'progressive') {
+            await expectEvent(EventType.AdManifestLoaded, 20);
+          }
```
Comment (rolandkakonyi) [inline integration_test/tests/advertisingTest.ts:62]: I experienced longer waiting times with the Android emulator, therefore added this extra expectation to stabilize the test. APPLIES TO THE OTHER OCCURRENCES TOO

File: integration_test/tests/advertisingTest.ts [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+            EventSequence(
+              EventType.AdBreakStarted,
+              EventType.AdStarted,
+              EventType.AdQuartile,
+              EventType.AdQuartile,
+              EventType.AdQuartile,
+              EventType.AdFinished,
+              EventType.AdBreakFinished
+            ),
+            18
```
Comment (zigavehovec) [inline integration_test/tests/advertisingTest.ts:73]: Is there a reason for this quite specific 18? Just a though, what if we increase the default timeout to account for Android emulators being a bit slower and set it to 20?

**Review by zigavehovec — APPROVED** [review]
LGTM!

File: integration_test/tests/advertisingTest.ts (hunk 2 above)
Comment (rolandkakonyi) [reply integration_test/tests/advertisingTest.ts:73]: I would not touch the default timeout as usually 10 seconds is pretty long already.
This is specific for ads on the Android emulator so belongs to the specific test.

**Pull request merged**